packages/compass-e2e-tests/helpers/commands/set-connect-form-state.ts

```diff
@@ -175,6 +175,9 @@ export async function setConnectFormState(
       Selectors.ConnectionFormTlsAllowInvalidCertificatesCheckbox
     );
   }
+  if (state.useSystemCA) {
```

Comment:
You probably want to put something in get-connect-form-state as well? maybe update connection-form.test.ts to toggle the field and check the state.

Reply:
Sure, done :+1: